packages/cli/src/commands/governance/approve.ts

```diff
@@ -28,33 +36,54 @@ export default class Approve extends BaseCommand {
       exclusive: ['proposalID'],
       description: 'Hash of hotfix proposal',
     }),
+    type: Flags.option({
+      description:
+        'Determines which type of hotfix approval (approver or security council) to use.',
+      dependsOn: ['hotfix'],
+      options: [HotfixApprovalType.APPROVER, HotfixApprovalType.SECURITY_COUNCIL],
+      multiple: false,
+      required: false,
+    })(),
   }
 
   static examples = [
     'approve --proposalID 99 --from 0x5409ed021d9299bf6814279a6a1411a7e866a631',
     'approve --proposalID 99 --from 0x5409ed021d9299bf6814279a6a1411a7e866a631 --useMultiSig',
     'approve --hotfix 0xfcfc98ec3db7c56f0866a7149e811bf7f9e30c9d40008b0def497fcc6fe90649 --from 0xCc50EaC48bA71343dC76852FAE1892c6Bd2971DA --useMultiSig',
+    'approve --hotfix 0xfcfc98ec3db7c56f0866a7149e811bf7f9e30c9d40008b0def497fcc6fe90649 --from 0xCc50EaC48bA71343dC76852FAE1892c6Bd2971DA --useMultiSig --type securityCouncil',
   ]
 
   async run() {
+    const checkBuilder = newCheckBuilder(this)
     const kit = await this.getKit()
     const res = await this.parse(Approve)
     const account = res.flags.from
     const useMultiSig = res.flags.useMultiSig
     const id = res.flags.proposalID
     const hotfix = res.flags.hotfix
+    const approvalType = res.flags.type
     kit.defaultAccount = account
     const governance = await kit.contracts.getGovernance()
     const governanceApproverMultiSig = useMultiSig
       ? await governance.getApproverMultisig()
       : undefined
+    const isCel2 = await this.isCel2()
+    const governanceSecurityCouncilMultiSig =
+      useMultiSig && isCel2 ? await governance.getSecurityCouncilMultisig() : undefined
     const approver = useMultiSig ? governanceApproverMultiSig!.address : account
 
-    const checkBuilder = newCheckBuilder(this)
-      .isApprover(approver)
-      .addConditionalCheck(`${account} is multisig signatory`, useMultiSig, () =>
-        governanceApproverMultiSig!.isOwner(account)
-      )
+    await addDefaultChecks(
```

Comment:
nitty but `add` as a verb doesnt seem right here. 

maybe `perform`

Reply:
my reasoning was the we just call `.add(...)` and actually run the checks later